### PR TITLE
Closes #64 - stop deploying SNAPSHOTS in the pipeline

### LINKED ISSUE #64: Remove SNAPSHOT Deployment from the Pipeline

### PR DESCRIPTION
**Pull request by ryzheboka**
### Thanks for your PR! Please fill out the following list :)

---

- [x] I put the ticket or multiple tickets in review
- [x] Commit message format → Closes #&lt;Issue Number&gt; - Your commit message.
- [ ] Sonarcloud link : https://sonarcloud.io/summary/new_code?id=ryzheboka_kadai&branch=KAD-64
- [x] No documentation update needed
- [ ] Link to PR with documentation update: \<add the link here>
- [x] No Release Notes needed
- [ ] Release Notes :

```

```

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -343,7 +343,7 @@ jobs:
   release_artifacts:
     runs-on: ubuntu-22.04
     name: Release artifacts to OSS Sonatype
-    if: github.repository == 'kadai-io/kadai' && ( startsWith(github.ref, 'refs/tags') || github.ref == 'refs/heads/master' ) && github.head_ref == ''
+    if: github.repository == 'kadai-io/kadai' && startsWith(github.ref, 'refs/tags/v') && github.head_ref == ''
```
Comment (CRoberto1926) [inline .github/workflows/continuous-integration.yml:346]: Do we need to specify "/v"? Isn't it enough to be a tag?

**Comment by CRoberto1926** [comment]
What about KadaiAdapter?

Should we modify also change_version.sh and/or commitPoms.sh to remove SNAPSHOT?

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (holgerhagen) [reply .github/workflows/continuous-integration.yml:346]: Probably not required. But it is a kind of QA to make sure, that all tags looks the same...

**Comment by ryzheboka** [comment]
I can do this for KadaiAdapter in the near future. I don't think it's necessary for the PRs to be there simulateously

**Comment by ryzheboka** [comment]
Do we want to have SNAPSHOT versions just without releasing them? Then, we wouldn't need to modify change_version.sh and/or commitPoms.sh

**Review by CRoberto1926 — APPROVED** [review]

**Pull request merged**